src/utils/send-files-stream.js

```diff
@@ -75,23 +75,10 @@ module.exports = (send, path) => {
 
     const qs = options.qs || {}
 
-    if (options['cid-version'] != null) {
-      qs['cid-version'] = options['cid-version']
-    } else if (options.cidVersion != null) {
-      qs['cid-version'] = options.cidVersion
-    }
-
-    if (options['raw-leaves'] != null) {
-      qs['raw-leaves'] = options['raw-leaves']
-    } else if (options.rawLeaves != null) {
-      qs['raw-leaves'] = options.rawLeaves
-    }
-
-    if (options.hash != null) {
-      qs.hash = options.hash
-    } else if (options.hashAlg != null) {
-      qs.hash = options.hashAlg
-    }
+    qs['cid-version'] = propOrProp(options, 'cid-version', 'cidVersion')
+    qs['raw-leaves'] = propOrProp(options, 'raw-leaves', 'rawLeaves')
+    qs['only-hash'] = propOrProp(options, 'only-hash', 'onlyHash')
+    qs.hash = propOrProp(options, 'hash', 'hashAlg')
```

Comment:
Please don't direct assign to `qs` which is a options that user pass to api.
This will cause error if user's options is read only